### PR TITLE
Button-2: Center image caption on frontend

### LINKED ISSUE #1725: Button 2: Image captions not centered
## Steps to replicate

When adding an image to a post in the Button 2 theme, the image caption looks centered in the editor, but when viewed live, it's aligned to the left.

1. Create a post on a site with the Button 2 theme.
2. Add an image and add a caption on the image and align it to center.
3. Publish the post.
4. View the post live to see the image caption location (reflected as centered in the editor, but to the bottom left of the image when viewed live).

## Result

The caption is aligned to the left, not center.

## Expected

I expected the caption alignment created in the editor would be the same with the location of the caption to be replicated on the live site.

It looks like there are other themes that have a similar issue, for example: #1377 

* User report: #17525948-hc

### PR DESCRIPTION
**Pull request by nielslange**
Fixes #1725 

<table>
<tr>
<td>Before:
<br><br>

#1725-before

</td>
<td>After:
<br><br>

#1725-after
</td>
</tr>
</table>

### CODE REVIEW

File: button-2/blocks.css [hunk 1]
```diff
@@ -20,7 +20,10 @@ Description: Used to style Gutenberg blocks
 
 /* Captions */
 
-[class^="wp-block-"] figcaption {}
+[class^="wp-block-"] figcaption {
```
Comment (michaeldcain) [inline button-2/blocks.css]: Let's go ahead and copy [the original caption styles](https://github.com/Automattic/themes/blob/master/button-2/style.css#L1916) for the theme here. And then update [the editor styles](https://github.com/Automattic/themes/blob/master/button-2/editor-blocks.css#L156) to reflect this.


```
font-family: Lora, Garamond, serif;
font-style: italic;
text-align: center;
```

Comment (nielslange) [reply button-2/blocks.css]: I've addressed the requested changes in https://github.com/Automattic/themes/pull/1726/commits/d82c13be435af125f9140355f60e3aad73925170

**Comment by nielslange** [comment]
> It looked like some extra commits snuck in from #1666 during the last push. I dropped them from the PR, and otherwise, this LGTM. Thanks!

@michaeldcain I reverted the rouge commits and only the affected files are part of this PR now. 😀

**Comment by michaeldcain** [comment]
> I reverted the rouge commits and only the affected files are part of this PR now.

@nielslange: FWIW, you can also do this by [rebasing the branch and dropping the unwanted commits](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History). I used `git rebase -i origin/master` to do this, and then force pushed it.

**Comment by michaeldcain** [comment]
This was deployed to WP.com in r55055-wpcom-themes

**Pull request merged**